TempleEight/docs/getting-started.md

```diff
@@ -102,234 +78,128 @@ ExampleService: service {
-
-For example, sending a `GET` request to the service with the entity's `id`, returns us the data.
-
-```
-~/Documents/temple-tutorial ❯❯❯ curl -X GET $KONG_ENTRY/api/example-service/e3621abe-7e40-11ea-9934-0242c0a88002
-{"id":"e3621abe-7e40-11ea-9934-0242c0a88002","foo":"v","bar":1689563322}
+### Reading an Entity
+To retrieve an existing entity from the service, invoke a GET request to the service's base URL, including the ID of the entity you want to examine:
+```bash
+~/temple-tutorial ❯❯❯ curl -X GET $KONG_ENTRY/api/example-service/e3621abe-7e40-11ea-9934-0242c0a88002
+{"id":"e3621abe-7e40-11ea-9934-0242c0a88002","foo":"Hello Temple!","bar":123}
 ```
+The response from the `READ` request is the JSON object representing the entity stored, or an error if it was not found.
 
-The full list of all generated endpoints is detailed in the [Ingress Guide](ingress).
-
-## Automated Testing
-
-Another tool contained in the Temple CLI is the automated testing tool.
-Temple will take the data model defined in the Templefile, as well as the generated sources and will automatically make requests that meet the format of the data, verifying that the returned values is as expected. 
-It can be a very useful way of end-to-end testing the application, verifying all components are functioning correctly.
-
+### Updating an Entity
+To update an existing entity, invoke a PUT request to the service's base URL, including the ID of the entity you want to update, as well as a JSON object of the new attributes to store:
+```bash
+~/temple-tutorial ❯❯❯ curl -X PUT $KONG_ENTRY/api/example-service/e3621abe-7e40-11ea-9934-0242c0a88002 -d {"foo": "Goodbye Temple!", "bar": 456}'
+{"id":"e3621abe-7e40-11ea-9934-0242c0a88002","foo":"Goodbye Temple!","bar":456}
 ```
-~/Documents/temple-tutorial ❯❯❯ temple test example.temple
-🐳 Spinning up Docker Compose infrastructure...
-🧪 Testing ExampleService service
-    ✅ ExampleService create
-    ✅ ExampleService read
-    ✅ ExampleService update
-    ✅ ExampleService delete
-🎉 Everything passed
-💀 Shutting down Docker Compose infrastructure...
+The response from the `UPDATE` request is the JSON object representing the new entity stored, or an error if it was not found.
+
+### Deleting an Entity
+To delete an existing entity, invoke a DELETE request to the service's base URL, including the ID of the entity you want to delete:
+```bash
+~/temple-tutorial ❯❯❯ curl -X DELETE $KONG_ENTRY/api/example-service/e3621abe-7e40-11ea-9934-0242c0a88002
+{}
 ```
+The response from the `DELETE` request is an empty JSON object.
 
 ## Next Steps
-
-This concludes our end to end walkthrough of the Temple ecosystem. For next steps, continue through the more advanced pages of the tutorial, or jump ahead:
+In this guide, we defined a simple data model, used Temple to generate a microservice, deployed the services locally and then performed some sample requests.
+Using this as a starting point, explore the full power of Temple:
 
```

Comment:
For some reason Github doesn't allow me to select the bullet points below this, I think because they're not in the diff. I suggest you change them to this:

```suggestion
* [Add authentication](guide/authentication)
* [Enforce access control](guide/access-control)
* [Reference other service entities with foreign keys](guide/foreign-keys)
* [Add business logic with hooks](guide/hooks)
* [Constrain stored values](guide/value-constraints)
* [Generate frontend APIs with OpenAPI](guide/openapi)
* [Add enumeration endpoints](guide/enumeration)
* [Augment with metrics](guide/metrics)
* [Add your own endpoints](guide/adding-endpoints)
* [Add your own database queries](guide/adding-dao-functions)
* [Remove endpoints from generation](guide/omitting-endpoints)
* [Restrict attributes to be server or client facing](guide/value-annotations)
* [Use alternative orchestration](guide/orchestration)
* [Make changes to your templefile and regenerate without losing your changes](guide/regeneration)
* [Test your project with Temple Test](guide/temple-test)
```

Please change the order of the bullets to make sense